### PR TITLE
add code lint check and fix code style

### PR DESCRIPTION
**Pull request by majian159**

### CODE REVIEW

File: dashboard/src/pages/System/Environment/index.tsx [hunk 1]
```diff
@@ -1,5 +1,3 @@
-import './index.less';
```
Comment (zzxwill) [inline dashboard/src/pages/System/Environment/index.tsx:1]: 多余的？

Comment (majian159) [reply dashboard/src/pages/System/Environment/index.tsx:1]: yeah, index.less is empty

**Comment by captainroy-hy** [comment]
@majian159 failed e2e test will be fixed in #811 , pls rebase after it's merged.

**Review by zzxwill — APPROVED** [review]
LGTM

**Pull request merged**